angular-app/src/app/models/mock/mock-appointments.ts

```diff
@@ -3,72 +3,73 @@ import { IUser, UserRole } from '../interfaces/IUser';
 import { MockParticipantList } from "./mock-users";
 
 const participants: IUser[] = MockParticipantList;
-const barber = { 
-  displayName:'danielr@chameleon.com', 
-  email: 'danielr@chameleon.com', 
+const barber = {
+  displayName:'danielr@chameleon.com',
+  email: 'danielr@chameleon.com',
   role: UserRole.TALENT
 };
 
 export const MockAppointmentList: IAppointment[] = [
```

Comment:
All the participants values can be empty array `[]` here after deleting the `IUser[]` type mentioned in `IAppointment.ts`